backend/core/database.py

```diff
@@ -786,10 +790,13 @@ def bootstrap_redis(self):
 
                 redis_pipeline = self.redis.pipeline()
                 for entry in entries:
-                    expire = int(time.time() - entry[1].timestamp())
+                    expire = max(
+                        int(entry[1].timestamp()) + 2 * 60 * 60 - int(time.time()), 60
```

Comment:
the max(...,60) is for the corner case that enough time has passed so that we are on the border of expiration; we set the expiry time in one minute or so in that case, just to be safe.